hpe3parclient/client.py

```diff
@@ -4439,3 +4439,141 @@ def targetInRemoteCopyGroupExists(
         except Exception:
             pass
         return False
+
+    def check_response(self, resp):
+        for r in resp:
+            if 'error' in str.lower(r) or 'invalid' in str.lower(r) or 'the schedule format' in str.lower(r):
```

Comment:
How does 'the schedule format' point to an error? Could you paste the exact command and the error output

Reply:
The schedule format is <minute> <hour> <dom> <month> <dow> or by @hourly @daily @monthly @weekly @monthly @yearly

Reply:
Please paste the entire command and its original output

Reply:
CSSOS-SSA04 cli% createsched "createsv snap volume_ansible_a" year sched
The schedule format is <minute> <hour> <dom> <month> <dow> or by @hourly @daily @monthly @weekly @monthly @yearly

Reply:
Add another check comparing the complete string. The reason for that is that the substring we are trying to find has the potential to appear in other places as well. We can't always be sure if this will only occur in this scenario

Reply:
done